apps/src/templates/sectionProgress/progressTables/ProgressTableView.jsx

```diff
@@ -162,12 +152,7 @@ class ProgressTableView extends React.Component {
 
   sortTableRows() {
     const comparator = keys => (a, b) =>
-      keys.reduce(
-        (result, key) =>
-          result || letterCompare(a.student[key] || '', b.student[key] || ''),
-        0
-      );
-
+      stringKeyComparator(keys)(a.student, b.student);
```

Comment:
This is slick! I was trying to think of a way to do this and couldn't quite get there.